@types/lesson.d.ts

```diff
@@ -27,5 +27,9 @@ export type Star = {
   studentId: number
   mentorId: number
   lessonId: number
+  lessonTitle: string
```

Comment:
We shouldn't have to be using from this file. Typescript definitions for data returned from graphql requests are in our codegen generated [graphql/index.tsx](https://github.com/garageScript/c0d3-app/blob/master/graphql/index.tsx) file